alpenhorn/io/Default.py

```diff
@@ -7,17 +7,144 @@
 explicitly specify `io_class` (as well as being used explicitly when `io_class`
 has the value "Default").
 """
+from __future__ import annotations
+from typing import TYPE_CHECKING
 
+import os
 import logging
+import pathlib
 
+from .. import util
 from .base import BaseNodeIO, BaseGroupIO
+from ..task import Task
+
+# The asyncs are over here:
+from ._default_asyncs import check_async
+
+if TYPE_CHECKING:
+    from ..archive import ArchiveFileCopy
 
 log = logging.getLogger(__name__)
 
 
 class DefaultNodeIO(BaseNodeIO):
     """A simple StorageNode backed by a regular POSIX filesystem."""
 
+    # I/O METHODS
+
+    def bytes_avail(self, fast: bool = False) -> int | None:
+        """bytes_avail: Return amount of free space (in bytes) of the node, or
+        None if that cannot be determined.
+
+        Does not account for space reserved via reserve_bytes().
+
+        Parameters
+        ----------
+        fast : bool
```

Comment:
```suggestion
        fast
```
I think the convention when using type annotations is to not include the type in the docstring. This would apply everywhere